packages/uniforms/src/createAutoField.tsx

```diff
@@ -14,6 +14,7 @@ import { useField } from './useField';
 export type AutoFieldProps = {
   component?: Component;
   name: string;
+  options?: { absoluteName?: boolean };
```

Comment:
@radekmie Does it make sense to also include `initialValue` here as well since it's also an option for `useField`?

Reply:
I'd still go with the `experimental_` prefix and allow only the `absoluteName` for now. `options` is an extremely popular prop, and hijacking it here would cause problems for a lot of projects.